socorro/signature/tests/test_utils.py

```diff
@@ -104,3 +109,140 @@ def test_collapse(function, expected):
         'exceptions': ['name omitted', 'IPC::ParamTraits', ' as ']
     }
     assert collapse(**params) == expected
+
+
+@pytest.mark.parametrize('function, expected', [
+    # C/C++
```

Comment:
@luser This is a bunch of test cases for the new "drop prefix and return type" code. Can you double-check that it's doing the right thing?

Reply:
This all looks sensible to me, thanks!